dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/README.md

```diff
@@ -26,14 +27,15 @@ Electronic consent is the same as written consent but collected via an electroni
 
 Note that, in case of electronic consent and SMS consent, it should be sent out to respondents before survey starts (ideally the day before)
 
-
 Most IRBs require that the entire written consent is shared with the participants and their signature is taken on another copy
 retained by the research team. 
+
 In cases that the written consent is too long to read out,
 we recommend reading out the oral consent to respondents and giving them a printed copy of the written consent.
 Note that the exact practice can vary based on the what has been agreed upon in the IRB approvals.
 It is essential to follow the guidelines set in the IRB.
 
+### Consent contents
 Informed consent must include, at a minimum, information about the authority for conducting the project,
```

Comment:
This might be the right place to list the requirements from the privacy policy: : (1) the type of Personal Data to be Processed; (2) the identity of the WBG Institution Processing the Personal Data; (3) whether the Personal Data is to be shared with Third Parties and the names or types of such Third Parties; (4) the specific purpose(s) for which the Personal Data is Processed; (5) the types of Processing activities that may be performed on the Personal Data; (6) how long the Personal Data is retained.

Reply:
are all 5 of these now explictly covered in the template?

Reply:
added in commit 656a021 and confirmed that all templates have all relevant information for all items except for 5. Unsure if the current set up gets at pointer 5.

Reply:
Also the SMS consent potentially doesn't cover all these points.

Reply:
i believe point 5 is covered by this section of the DPO's sample: "Any response that you provide to my questions will be kept confidential. No specific information about you will be shared with third parties. A compilation of all responses from
[name community/country] will be provided to the World Bank ..."

Reply:
> Also the SMS consent potentially doesn't cover all these points.

it will have to in order for us to be able to legally use the data from SMS surveys